data/schema/planarloc.json

```diff
@@ -0,0 +1,17 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://raw.githubusercontent.com/swarm-game/swarm/main/data/schema/planarloc.json",
+    "title": "Cosmic location",
+    "description": "Planar location",
```

Comment:
Maybe expand on this description a bit, *e.g.* "x and y coordinates of a location in a particular world" or something like that.